lib/perl/Genome/InstrumentData/AlignmentResult.pm

```diff
@@ -1629,9 +1629,112 @@ sub verify_alignment_data {
 }
 
 sub alignment_bam_file_paths {
+    return glob(shift->output_dir . "/*.bam");
+}
+
+sub recreated_alignment_bam_file_paths {
+    my $self = shift;
+
+    my @bams = glob($self->output_dir . "/*.bam");
+    if (@bams) {
+        return @bams;
+    }
+
+    my $recreated_bam = File::Spec->join($self->output_dir, 'all_sequences.bam');
+    my $merged_bam    = $self->get_merged_bam_to_revivify_per_lane_bam;
```

Comment:
I like `revivify` as a word (It's almost as if we're alignment necromancers!)  But it's weird that it's only used here when `recreate` is used everywhere else.